integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/bootstrap/ConfiguringPropertiesInSearchMappingBuilderIT.java

```diff
@@ -0,0 +1,123 @@
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.pojo.bootstrap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.lang.invoke.MethodHandles;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.CompletableFuture;
+
+import org.hibernate.search.engine.backend.analysis.AnalyzerNames;
+import org.hibernate.search.engine.backend.types.Projectable;
+import org.hibernate.search.engine.cfg.ConfigurationPropertySource;
+import org.hibernate.search.engine.cfg.EngineSettings;
+import org.hibernate.search.engine.cfg.spi.AllAwareConfigurationPropertySource;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.standalone.cfg.StandalonePojoMapperSettings;
+import org.hibernate.search.mapper.pojo.standalone.cfg.spi.StandalonePojoMapperSpiSettings;
+import org.hibernate.search.mapper.pojo.standalone.mapping.CloseableSearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMappingBuilder;
+import org.hibernate.search.mapper.pojo.standalone.mapping.StandalonePojoMappingConfigurationContext;
+import org.hibernate.search.mapper.pojo.standalone.mapping.StandalonePojoMappingConfigurer;
+import org.hibernate.search.mapper.pojo.work.IndexingPlanSynchronizationStrategyNames;
+import org.hibernate.search.util.impl.integrationtest.common.bean.ForbiddenBeanProvider;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
+import org.hibernate.search.util.impl.integrationtest.common.stub.backend.index.StubSchemaManagementWork;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+public class ConfiguringPropertiesInSearchMappingBuilderIT {
+
+	@Rule
+	public BackendMock backend = new BackendMock();
+
+	@Test
+	public void indexAndSearch() throws IOException {
+		ConfigurationPropertySource propertyFile = null;
+		try ( Reader properties = new InputStreamReader(
+				getClass().getClassLoader().getResourceAsStream( "standalone-test.properties" ),
+				StandardCharsets.UTF_8
+		) ) {
+			propertyFile = AllAwareConfigurationPropertySource.fromPropertyFile( properties );
+		}
+
+		SearchMappingBuilder builder = SearchMapping.builder( MethodHandles.lookup() )
+				.property( StandalonePojoMapperSpiSettings.BEAN_PROVIDER, new ForbiddenBeanProvider() )
+				.properties( propertyFile )
+				.property(
+						StandalonePojoMapperSettings.INDEXING_PLAN_SYNCHRONIZATION_STRATEGY,
+						IndexingPlanSynchronizationStrategyNames.SYNC
+				)
```

Comment:
No sure what's the purpose of this and the corresponding `async` in the property file.

Reply:
The idea was to check that properties are getting overridden, but since I've lost the index and search part of the test somewhere ... well the test looks different now anyway 😃